### PR TITLE
Fix KT-002-PACKIT-SERVICE-4YN

### PR DESCRIPTION
**Pull request by TomasTomecek**
TODO:

- [x] Write new tests or update the old ones to cover new functionality.

Two fixes here: check commit messages :)

Fixes KT-002-PACKIT-SERVICE-4YN

### CODE REVIEW

**Comment by TomasTomecek** [comment]
14 minutes later... queued :<

**Comment by TomasTomecek** [comment]
recheck

File: packit_service/worker/checker/vm_image.py [hunk 1]
```diff
@@ -82,15 +82,21 @@ def pre_check(
...
-                    build.target == job_config.copr_chroot
-                    and build.status == BuildStatus.success
-                ):
-                    return True
+            for copr_package in build.built_packages:
+                copr_package_name = copr_package["name"]
+                if copr_package["arch"] == "src":
+                    # we care about built packages, not SRPM
+                    continue
+                if copr_package_name in self.job_config.packages:
```
Comment (majamassarini) [inline packit_service/worker/checker/vm_image.py:90]:
```suggestion
```

 I am testing your PR with my local test and now it does not pass anymore the copr build check (which is awkward but was passing with the old code 😅). 
The problem is this line.
In my project the `copr_project_name` is `python-knx-stack` and the `job_config.packages` is `[knx-stack]`. 
If I remove this line everything seems ok. Do you think we can safely remove it?

Comment (TomasTomecek) [reply packit_service/worker/checker/vm_image.py]: we have discussed on Matrix

agreed this is not a correct solution

**Comment by TomasTomecek** [comment]
recheck

**Review by majamassarini — APPROVED** [review]
Thanks! Now is working :)

**Pull request merged**